### PR TITLE
Validate CREATE SERVER options

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

File: server/src/main/java/io/crate/fdw/ForeignDataWrapper.java [hunk 1]
```diff
@@ -34,6 +36,10 @@
 
 public interface ForeignDataWrapper {
 
+    default List<Setting<?>> optionalServerOptions() {
```
Comment (mfussenegger) [inline server/src/main/java/io/crate/fdw/ForeignDataWrapper.java:39]: Didn't add `mandatoryServerOptions` yet because there aren't any

**Review by matriv — COMMENTED** [review]
Thx, looks good, I posted a thought about a default connection string though, please let me know.

File: server/src/main/java/io/crate/fdw/JdbcForeignDataWrapper.java [hunk 2]
```diff
@@ -54,12 +55,19 @@ final class JdbcForeignDataWrapper implements ForeignDataWrapper {
 
     private final InputFactory inputFactory;
     private final Settings settings;
+    private final Setting<String> urlSetting = Setting.simpleString("url","jdbc:postgresql://localhost:5432/");
```
[the remark this answers is not in the document]
Comment (mfussenegger) [reply server/src/main/java/io/crate/fdw/JdbcForeignDataWrapper.java:58]: I don't really see a downside to it either? Worst case is it is the wrong value and the connection doesn't work. In which case you need to specifiy it explicitly.

**Comment by mfussenegger** [comment]
@matriv added a fixup to make it mandatory. Can you check again?

**Review by matriv — APPROVED** [review]
thank you! LGTM

**Pull request merged**